### DISCUSSION TITLE
replays from lazer to stable, key inputs get messed up

### DISCUSSION THREAD

**Discussion by Buzzboss** [ppy/osu#27047] [Ideas] [answered]
i saved a replay in lazer then opened the replay file and it opened in stable everything was fine but the key inputs are swaped with mouse inputs on the key overlay

https://github.com/ppy/osu/assets/110465394/d7463d2f-c9e1-44be-9c63-d043cef83ea2

**Comment by bdach** [comment, marked as answer]
There is no such thing as "mouse inputs" on lazer. You get two arbitrary keys to bind to two click actions, and that's it.
